src/platform/Ameba/Logging.cpp

```diff
@@ -50,18 +59,21 @@ void LogV(const char * module, uint8_t category, const char * msg, va_list v)
     switch (category)
     {
     case kLogCategory_Error:
-        printf("%s %s\r\n", tag, formattedMsg);
+        if (LogLevel >= AmebaLogLevel::kError)
```

Comment:
If using enum class, you probably cannot compare unless you cast to the underlying type (src/lib/support/TypeTraits.h has to_underlying)

Reply:
not very familar with this, can you review my changes? Thanks.

Reply:
@andy31415 , im resolving this but please reopen it if there's any issues